src/main/java/be/cytomine/client/models/UploadedFile.java

```diff
@@ -68,6 +70,14 @@ public UploadedFile(String originalFilename, String realFilename, String path, L
         }
     }
 
+    public AttachedFile uploadAttachedFile(String file, String domainClassName, Long domainIdent) throws CytomineException {
```

Comment:
AttachedFile is different from UploadedFile

Reply:
done